packages/accordion/stories.js

```diff
@@ -0,0 +1,208 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { createRef, useState } from 'react';
+
+import { storiesOf } from '@storybook/react';
+import { boolean, number, withKnobs } from '@storybook/addon-knobs';
+
+import { AccordionContainer, useAccordion, SectionContainer, useSection } from './src';
+
+storiesOf('Accordion Container', module)
+  .addDecorator(withKnobs)
+  .add('useAccordion', () => {
+    const size = number('Sections', 5, { range: true, min: 1, max: 9 });
+    const sections = Array(size)
+      .fill()
+      .map(() => createRef());
+
+    const Accordion = ({ expandable = true, collapsible = true } = {}) => {
+      const { getSectionProps } = useAccordion({
+        expandedSections: [sections[0]],
+        expandable,
+        collapsible
+      });
+
+      return (
+        <div style={{ width: 300 }}>
+          {sections.map((section, index) => {
+            const sectionProps = getSectionProps({ section, idPrefix: index });
```

Comment:
[discussion] The usage of a prop-getter to get additional prop-getters is confusing to me here. This differs from the other implementations. Could you go into some detail here?

Reply:
Yeah, this is a main variation from existing container patterns. This was that area you and I discussed where an accordion does not express a container for individual sections. But the sections have elements (header, trigger, panel) that need to relate to each other. I'm sure there are other ways to structure (I've tried several). This ended up making the most sense to me, given the goals listed in the PR description. I'm open to other approaches.

Reply:
Perhaps it should just be named `getSection` to indicate that it isn't intended to be spread?

Reply:
Got rid of `useSection` per Ryan's comment.